### PR TITLE
Fix some errors on my machine.

### PR DESCRIPTION
**Pull request by l3gacyb3ta**
Hi! It's me from email. I got some errors while building, but this has fixed it all.

### CODE REVIEW

File: spor.c [hunk 1]
```diff
@@ -647,10 +647,12 @@ inline static void executeInstr(Instr instr) {
       if(WS_POP()) return;
       // intentional fallthrough
     case RET:
+    {
```
Comment (vitiral) [inline spor.c:650]: These should not be necessary. Can you try without them and let me know the error that happens when they are removed?

**Review by vitiral — APPROVED** [review]
Thanks! Could you add the following in your commit message (see README.md "Contributing" section):

```
I <author> assent to license this and all future contributions to this project
under the dual licenses of the UNLICENSE or MIT license listed in the
`UNLICENSE` and `README.md` files of this repository.
```

FYI the unlicense means this code is public domain (although I will personally always attribute individuals who help, nobody else has to).

**Comment by l3gacyb3ta** [comment]
I Arcade Wise assent to license this and all future contributions to this project
under the dual licenses of the UNLICENSE or MIT license listed in the
`UNLICENSE` and `README.md` files of this repository.

File: spor.c (hunk 1 above)
Comment (vitiral) [reply spor.c:650]: Oh! Weird that my compiler didn't throw this error. It's complaining about the variable declaration.

Replace
```
U4 cont = Stk_pop(&env.cs);
```

With
```
r = Stk_pop(&env.cs);
```

Then replace the uses of `cont` with `r`.

Comment (vitiral) [reply spor.c:650]: I didn't realize this could be fixed by using `{ ... }`. That's good to know!

Comment (l3gacyb3ta) [reply spor.c:650]: It works!

**Comment by l3gacyb3ta** [comment]
Ok! I have fixed up the git history so, it should be cleaner now.

**Review by vitiral — APPROVED** [review]

**Comment by l3gacyb3ta** [comment]
feel free to merge!

**Comment by vitiral** [comment]
Merged in `3452f3f35425917b4068f80f92aed33f6dae4985`

**Pull request closed without merging**